core/src/main/java/org/kohsuke/stapler/export/Property.java

```diff
@@ -378,7 +382,7 @@ void commit(DataWriter w) throws IOException {
                     w.name((String) step.args[0]);
                     break;
                 case valuePrimitive:
-                    w.valuePrimitive(step.args[0]);
+                    valuePrimitive(step.args[0]);
```

Comment:
This looks like a breaking change.  Why was it made?
```suggestion
                    w.valuePrimitive(step.args[0]);
```